components/component_style/badge.css

```diff
@@ -0,0 +1,68 @@
+.badge-container {
+    display: flex;
+    align-items: center;
+    margin-top: var(--md-spacing);
+    position: relative;
+}
+
+.badge-container span {
+    margin-left: var(--xs-spacing);
+}
+
+.badge {
+    font-weight: var(--bold-font-weight);
+    padding: var(--xs-spacing) var(--sm-spacing);
+    border-radius: var(--default-border);
+}
+
+.badge-primary-contained {
+    background-color: var(--primary-color-dark-shade3);
+    color: white;
+    border: 2px solid var(--primary-color-dark-shade3);
+}
+
+.badge-primary-outlined {
+    color: var(--primary-color-dark-shade3);
+    border: 2px solid var(--primary-color-dark-shade3);
+    background-color: white;
+}
+
+.badge-secondary-contained {
+    background-color: var(--secondary-color);
+    color: white;
+    border: 2px solid var(--secondary-color);
+}
+
+.badge-secondary-outlined {
+    color: var(--secondary-color);
+    border: 2px solid var(--secondary-color);
+    background-color: white;
+}
+
+.badge-round {
+    border-radius: var(--round-border);
+}
+
+.sm-badge {
+    padding: 0rem var(--xs-spacing);
+    font-size: var(--sm-font);
+}
+
+.status-badge {
+    min-height: 1rem;
+    min-width: 1rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    bottom: 10px;
+    left: 10px;
+    cursor: pointer;
+}
+
+.icon-badge {
+    background-color: white;
```

Comment:
you can also put white in var, to maintain consistency.Although i have done similar mistakes but try to remember from next time.

Reply:
will do, thanks for suggestion